src/core/constants.js

```diff
@@ -274,8 +274,14 @@ export const ADDONS_REVIEW = 'Addons:Review';
 export const THEMES_REVIEW = 'Personas:Review';
 // Can view statistics for all addons, regardless of privacy settings.
 export const STATS_VIEW = 'Stats:View';
-// Can edit all collections.
-export const COLLECTIONS_EDIT = 'Collections:Edit';
```

Comment:
Are you sure we don't need this anymore? It seems like it was intended for superuser collection editors that aren't admins.

Reply:
We got rid of that permission, which is the reason for this issue. Only the two permissions being implemented in this issue should allow someone who is not a global admin to edit a collection which they do not own. This change has already been made on the back end (disabling the `Collections:Edit` permission.